src/contexts/W3iContext/hooks/pushHooks.ts

```diff
@@ -47,12 +47,13 @@ export const usePushState = (w3iProxy: Web3InboxProxy, proxyReady: boolean, dapp
 
   const handleRegistration = useCallback(
     async (key: string) => {
+      console.log({ pushClient, key, pushEnabled: uiEnabled.notify })
       if (pushClient && key && uiEnabled.notify) {
         try {
           const identityKey = await pushClient.register({ account: `eip155:1:${key}` })
+          refreshPushState()
           setRegisterMessage(null)
           setRegistered(identityKey)
-          refreshPushState()
```

Comment:
What's the reasoning for this?

Reply:
I looked into how `chatHooks` handle it and I used the same order, curious if it matters ?